xbmc/platform/win32/WinMain.cpp

```diff
@@ -15,6 +15,7 @@
 #include "platform/xbmc.h"
 #include "threads/Thread.h"
 #include "utils/CharsetConverter.h" // Required to initialize converters before usage
+#include "utils/StringUtils.h"
```

Comment:
I'm not sure of include `StringUtils` here: is a big code and if produces crash at this point is hard to debug as main kodi logs, mini dump, etc is not yet initialized. Maybe use simple wsprintf or system equivalent for formatting these strings.

Reply:
ok. took a while to find the correct variation of printf for safe buffer allocation but I have something.